recipes-core/images/iot2050-image-base.bb

```diff
@@ -56,3 +56,10 @@ python image_postprocess_restore_sources_list () {
     else:
         bb.note('No need to restore sources for mainline packages')
 }
+
+# Make the .wic.img symlink to the .wic file for better backward compatibility
+do_deploy() {
+    echo "Linking wic img"
+    rm -f ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
+    ln -s ${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
```

Comment:
Should the full path be used here for .wic

Reply:
No, the full path could not be determined, it is different across the build env and the host env, neither across different host env.

Reply:
I mean 
```
ln -s ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
```

Reply:
> I mean
> 
> ```
> ln -s ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
> ```

The `${DEPLOY_DIR_IMAGE}` is not the correct host path if the image is built with docker.